minio-operator.yaml

```diff
@@ -196,7 +196,7 @@ spec:
       containers:
         - name: minio-operator
           image: minio/k8s-operator:2.0.6
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```

Comment:
shouldn't this be? IfNotPresent?

Reply:
I don't have an opinion other than it helps with debugging locally.